docs/add-on/guides/doorbell.md

```diff
@@ -137,13 +137,34 @@ Look <a href="https://wiki.asterisk.org/wiki/display/AST/ConfBridge+Configuratio
 
 ### Redial
 
-You can also try to redial until someone registers and picks up.
+This is a bit more complicated dialplan, but basicly it tries to call, and if it hangsup because the endpoint isn't registered it tries again after 1 second. And it keeps trying until it tried 60 times.
 
-:::caution
+When dialing, you can use the asterisk integration to trigger a automation that sends a notification to your phone to open the sip card.
```

Comment:
Is this notification system documented somewhere? I think it would be really nice to cover this gap. There are some people who really don't want to tinker on it.

Reply:
NVM, I think this is the same as:

- #8